decko/rails/controllers/card_controller.rb

```diff
@@ -55,7 +55,7 @@ def asset
 
   def setup
     request.format = :html unless params[:format] # is this used??
-    Card::Machine.refresh_script_and_style
+    Card::Machine.refresh_script_and_style unless params[:explicit_file]
```

Comment:
I noticed that without the assets symlink we check all script and style files for updates on every file request

Reply:
👍